lib/cartopy/tests/mpl/test_caching.py

```diff
@@ -122,7 +122,8 @@ def test_contourf_transform_path_counting():
     gc.collect()
     initial_cache_size = len(cgeoaxes._PATH_TRANSFORM_CACHE)
 
-    with mock.patch('cartopy.mpl.patch.path_to_geos') as path_to_geos_counter:
+    with mock.patch('cartopy.mpl.patch.path_to_shapely',
+                    wraps=cartopy.mpl.patch.path_to_shapely) as path_to_shapely_counter:
```

Comment:
This was failing because `project_geometry` received a `MagicMock` that it didn't know what to do with.  I'm confused why that didn't also happen previously with `path_to_geos`.